File: apenas java/calculoSimples.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double total = 0;
        for (int i = 0; i < 2; i++) {
            sc.nextInt();
            int quantidade = sc.nextInt();
            double preco = sc.nextDouble();
            total += quantidade * preco;
        }
        String str = String.format("%.2f",total);
        System.out.println("VALOR A PAGAR: R$ " + str);
        sc.close();
    }
}

File: apenas java/calculoSimples.c
#include <stdio.h>
int main() {
    double total = 0;
    for (int i = 0; i < 2; i++) {
        int quantidade;
        scanf("%d",&quantidade);
        scanf("%d",&quantidade);
        double preco;
        scanf("%lf",&preco);
        total += quantidade * preco;
    }
    printf("VALOR A PAGAR: R$ %0.2f\n",total);
    return 0;
}
